src/components/Footer.js

```diff
@@ -0,0 +1,11 @@
+import { render } from '../shared/dom';
+
+const template = `
+    <div>
```

Comment:
<footer /> tag

Reply:
You can add:
<footer>
<p>Copyright © 2021 CodersCamp</p>
</footer>